jx_sqlite/aggs_table.py

```diff
@@ -360,12 +360,21 @@ def _get(row):
                     column_number = len(outer_selects)
 
                     for json_type, sql in details.sql.items():
-                        concat_sql.append("GROUP_CONCAT(QUOTE(DISTINCT(" + sql + ")))")
+                        concat_sql.append("GROUP_CONCAT(DISTINCT(" + sql + "))")
 
                     if len(concat_sql) > 1:
                         concat_sql = "CONCAT(" + ",".join(concat_sql) + ") AS " + _make_column_name(column_number)
                     else:
                         concat_sql = concat_sql[0] + " AS " + _make_column_name(column_number)
+                    
+                    query_tables = [t["nest"] for t in tables]
```

Comment:
I do believe `tables` is `wrap`ed, so you can `query_tables = tables.nest` instead